IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -36,7 +36,7 @@ In total we will have a talk every Monday. The times for the talks are:
 9:00 **17:00** 18:00 |
 
 
-If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol labs dot ai.
+If you have a talk you want to give, [sign up here](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898) **or** please email me at portia @ protocol dot ai.
```

Comment:
This paragraph should link to the steps below, which are more complex than just adding ourselves to a spreadsheet.

Reply:
I had second thoughts about the spreadsheet and I wondering if it would be a better experience if folks just emailed their talk ideas to me and I'll add it to the spreadsheet myself.